tools/@aws-cdk/pkglint/lib/readme-contents.ts

```diff
@@ -66,9 +66,9 @@ export function cfnOnlyReadmeContents(options: LibraryReadmeOptions) {
     ...(options.alphaPackageName ? [
       '',
       '> The construct library for this service is in preview. Since it is not stable yet, it is distributed',
-      '> as a separate package so that you can pin its version independently of the rest of the CDK. See the package named:',
+      '> as a separate package so that you can pin its version independently of the rest of the CDK. See the package:',
       '>',
-      `>     ${options.alphaPackageName}`,
+      `> <span class="package-reference">${options.alphaPackageName}</span>`,
```

Comment:
```suggestion
      `> <span class="alpha-package-reference">${options.alphaPackageName}</span>`,
```

Reply:
I don't mind adding this, but I'm also not sure why it would matter whether the thing we're referring to is an alpha package or not.

Are you saying the following should not work:

```
<span class="alpha-package-reference">@aws-cdk/core</span>
```

(Leave alone whether we would emit that or not...)

Reply:
oh I see that is a good point.